community/posters.md

```diff
@@ -0,0 +1,67 @@
+# Event Poster Guidelines
+See also: The [design](https://github.com/kossiitkgp/design/) repository. Please add all poster designs to this repository.
+
+### Poster Design Guidelines
+#### Size
+- For printable posters, the size must be **11.5x17.5 inches** to be printed properly without any borders and it must be in portrait orientation.
```

Comment:
why should it always be portrait ?
We had posters in landscape orientation and it worked fine. Also, social media is more flexible for landscape orientation compared to portrait orientation...

Also, why no borders? 
Having some margin would be good, for pasting purposes

Reply:
Ig portrait posters are better for sticking (space efficient)

Reply:
> Also, why no borders?
Having some margin would be good, for pasting purposes

@proffapt?

Reply:
don't make limitations for portrait, whatever you do make sure the dimensions are the one mentioned. And don't force only portrait, doesn't make sense.

> Ig portrait posters are better for sticking (space efficient)

non-sense.

Reply:
there will never be lack of space.

Reply:
Thik hai bhai

Reply:
lekin kisi to advisor ne bola tha portrait banane ke liye ab yaad nahi kaun

Reply:
![image](https://github.com/kossiitkgp/docs/assets/34770591/3300d644-7b16-4607-9f59-db86777fabba)

Portrait preferred kar deta hoon